Tests/test_dlrkwarg.py

```diff
@@ -0,0 +1,85 @@
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the Apache 2.0 License.
+# See the LICENSE file in the project root for more information.
+
+import unittest
+
+from iptest import IronPythonTestCase, is_cli, is_mono, is_netcoreapp, is_posix, long, run_test, skipUnlessIronPython
+
+if is_cli:
+    import System
+    from iptest.ipunittest import load_ironpython_test
+    load_ironpython_test()
+    from IronPythonTest import DefaultParams, Variadics
+
+
+@skipUnlessIronPython()
+class DlrKwargTest(IronPythonTestCase):
+    def test_defaults(self):
+        self.assertEqual(DefaultParams.FuncWithDefaults(1100, z=82), 1184)
+
+    def call_kwargs_func(self, func):
+        self.assertEqual(func(), 0)
+        self.assertEqual(func(arg1='arg1'), 1)
+        self.assertEqual(func(arg1='arg1', arg2='arg2'), 2)
+        self.assertEqual(func(arg1='bad', arg2='arg2'), 'arg1')
+        self.assertEqual(func(arg1='arg1', arg2='bad'), 'arg2')
```

Comment:
Should we maybe also test the ** syntax? Or is that covered elsewhere?
```Python
func(**{"arg1": 'arg1'})
```

Reply:
I thought `test_function.FunctionTest('test_kwarg3')` covered it enough, but it doesn't hurt to add it here too.